src/rebar_file_utils.erl

```diff
@@ -43,6 +43,9 @@
          touch/1,
          path_from_ancestor/2,
          canonical_path/1,
+         absolute_path/1,
+         normalized_path/1,
+         normalized_path/2,
```

Comment:
hm, I don't like this one being exported. We could add `normalized_relative_path/1` or something and export only that, and delegate the internals to `normalized_path/2` which would be unexported and respect isolation better.

Reply:
Gotcha. Just to make sure: move also `normalized_relative_path/1` to `rebar_file_utils` and export that one instead of `normalized_path/2` while keeping `normalized_path/1` exported?

Reply:
yep